nsv13/code/modules/overmap/fighters/fighters.dm

```diff
@@ -1165,11 +828,134 @@ How to make fuel:
 	flight_state = NO_IGNITION
 	return FALSE
 
+/obj/structure/overmap/fighter/proc/set_master_caution(state)
+	var/master_caution_switch = state
+	if(master_caution_switch)
+		to_chat(usr, "<span class='warning'>WARNING: Master caution.</span>")
+		relay('nsv13/sound/effects/fighters/master_caution.ogg', null, loop=TRUE, channel=CHANNEL_BUZZ)
+		master_caution = TRUE
+	else
+		stop_relay(CHANNEL_BUZZ)
+		master_caution = FALSE
+
+/obj/structure/overmap/fighter/proc/get_fuel()
+	var/obj/item/fighter_component/fuel_tank/ft = get_part(/obj/item/fighter_component/fuel_tank)
+	if(!ft)
+		return 0
+	var/return_amt = 0
+	for(var/datum/reagent/aviation_fuel/F in ft.reagents.reagent_list)
+		if(!istype(F))
+			continue
+		return_amt += F.volume

```

Comment:
I'm sure there's a reason to prefer this over 
if(istype(F))
    return_amt += F.volume

but I'm not sure what it is